tests/liteclient_test.cc

```diff
@@ -170,38 +172,41 @@ class TufRepoMock {
   Uptane::Target latest_;
 };
 
+/**
+ * Class DeviceGatewayMock
+ *
+ */
 class DeviceGatewayMock {
  public:
   static std::string RunCmd;
+
  public:
-  DeviceGatewayMock(const OSTreeMock& ostree_mock,
-                    const TufRepoMock& tuf_repo_mock):
-                    ostree_mock_{ostree_mock},
-                    tuf_repo_mock_{tuf_repo_mock},
-                    port_{TestUtils::getFreePort()},
-                    url_{"http://localhost:" + port_},
-                    req_headers_file_{tuf_repo_mock_.path() + "/headers.json"},
-                    process_{RunCmd, "--port", port_, "--ostree", ostree_mock_.path(), "--tuf-repo", tuf_repo_mock_.path(), "--headers-file", req_headers_file_} {
+  DeviceGatewayMock(const OSTreeRepoMock& ostree, const TufRepoMock& tuf)
+      : ostree_{ostree},
+        tuf_{tuf},
+        port_{TestUtils::getFreePort()},
+        url_{"http://localhost:" + port_},
+        req_headers_file_{tuf_.getPath() + "/headers.json"},
+        process_{RunCmd,           "--port",         port_, "--ostree", ostree_.getPath(), "--tuf-repo", tuf_.getPath(),
```

Comment:
looks like additional empty spaces?

Reply:
I didnt do those...must have been the clang format. should I remove them?